src/javascript/app/pages/trade/purchase.js

```diff
@@ -134,6 +134,19 @@ const Purchase = (() => {
                                 additional_message = localize('Try our other markets.');
                             }
                             message = `${error.message}. ${additional_message}`;
+                        } else if (/ClientUnwelcome/.test(error.code) && /gb/.test(State.getResponse('authorize.country'))) {
+                            let message_text = '';
+                            let additional_message = '';
+
+                            if (!Client.hasAccountType('real') && /^VRTC/.test(State.getResponse('authorize.loginid'))) {
+                                message_text = localize('Please complete the [_1]Real Account form[_2] to verify your age as required by the <strong>UK Gambling</strong> Commission (UKGC).', [`<a href='${urlFor('new_account/realws')}'>`, '</a>']);
```

Comment:
do we want to use placeholders or keep the `<strong>` tag in the translation string?

Reply:
if I am not mistaken, we want to avoid using too many placeholders since it will make confusion to the translator?

Reply:
let me keep it as placeholders instead